src/server.js

```diff
@@ -84,13 +93,17 @@ module.exports = function() {
 	manager.sockets = sockets;
 
 	const protocol = config.https.enable ? "https" : "http";
-	const host = config.host || "*";
 
 	log.info(`The Lounge ${colors.green(Helper.getVersion())} is now running \
 using node ${colors.green(process.versions.node)} on ${colors.green(process.platform)} (${process.arch})`);
 	log.info(`Configuration file: ${colors.green(Helper.CONFIG_PATH)}`);
-	log.info(`Available on ${colors.green(protocol + "://" + host + ":" + config.port + "/")} \
-in ${config.public ? "public" : "private"} mode`);
+	if (typeof config.host !== "undefined" && config.host.includes("unix:")) {
+		log.info(`Available on socket ${colors.green(config.host)} \
```

Comment:
The same message can be used for both if you just set `protocol` to `unix` here for the message template.